### PR TITLE
EPMRPP-40113. Cumulative trend chart. Incorrect minimized illustratio…

### PR DESCRIPTION
**Pull request by ElenaRomanchuk**
…n via "Print"

### CODE REVIEW

File: app/src/components/widgets/charts/cumulativeTrendChart/cumulativeTrendChart.jsx [hunk 1]
```diff
@@ -192,6 +199,13 @@ export class CumulativeTrendChart extends PureComponent {
     });
   };
 
+  toggleControlsVisibilityVisibility = () => {
```
Comment (AmsterGet) [inline app/src/components/widgets/charts/cumulativeTrendChart/cumulativeTrendChart.jsx]: `toggleControlsVisibility`

File: app/src/components/widgets/charts/cumulativeTrendChart/legend/cumulativeChartLegend.jsx [hunk 2]
```diff
@@ -52,6 +54,7 @@ export class CumulativeChartLegend extends PureComponent {
       clearAttributes,
       userSettings,
       isChartDataAvailable,
+      hideControls,
     } = this.props;
     const { defectTypes, showTotal, separate, percentage } = userSettings;
     return (
```
Comment (AmsterGet) [inline app/src/components/widgets/charts/cumulativeTrendChart/legend/cumulativeChartLegend.jsx]: Shouldn't you pass `hideControls` prop to `CumulativeChartBreadcrumbs`?

File: app/src/components/widgets/charts/cumulativeTrendChart/cumulativeTrendChart.jsx [hunk 3]
```diff
@@ -63,6 +66,10 @@ export class CumulativeTrendChart extends PureComponent {
     }
   }
 
+  componentWillUnmount() {
+    this.props.observer.unsubscribe('widgetResized', this.toggleControlsVisibility);
```
Comment (AmsterGet) [inline app/src/components/widgets/charts/cumulativeTrendChart/cumulativeTrendChart.jsx]: `Cumulative trend chart` can't be resized.
See the comments from https://jiraeu.epam.com/browse/EPMRPP-39637

**Comment by ElenaRomanchuk** [comment]
image

**Pull request merged**